IncBackups/IncBackupsControl.py

```diff
@@ -283,17 +276,17 @@ def localFunction(self, backupPath, type, restore=None, delete=None):
     ## Last argument delete is set when the snapshot is to be deleted from this repo, when this argument is set, any preceding argument is not used
 
     def sftpFunction(self, backupPath, type, restore=None, delete=None):
```

Comment:
This function was changed a lot to accomodate rclone (specifically onedrive right now, but that only refers to the name of the rclone remote that I created ("onedrive"). You could test this with any provider, so long as you call the rclone remote "onedrive".